warehouse/templates/manage/account/recovery_codes-provision.html

```diff
@@ -0,0 +1,76 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+{% extends "manage/manage_base.html" %}
+
+{% set user = request.user %}
+{% set title = gettext("2FA Recovery Codes") %}
+
+{% set active_tab = 'account' %}
+
+{% block title %}
+  {{ title }}
+{% endblock %}
+
+{# Hide mobile search on manager pages #}
+{% block mobile_search %}{% endblock %}
+
+{% block content %}
+<div class="horizontal-section">
+  <div class="site-container">
+    <div class="col-half">
+      <h1 class="page-title page-title--wsubtitle heading-wsubtitle__heading">{{ title }}</h1>
+      <p>If you lose access to your authentication application or security key(s), you’ll need to use one of these recovery codes to log into your PyPI account. Each code can only be used <strong>once</strong>.</p>
+      <p>These codes should <strong>only</strong> be used for account recovery, not for typical logins.</p>
+      <p><strong>Keep these somewhere safe</strong>. If you lose your authentication application or security key(s) and do not have access to these recovery codes, you may permanently lose access to your PyPI account!</p>
+    </div>
+    <div class="col-half">
+      <div>
+        <h2>{% trans %}Save your Recovery Codes{% endtrans %}</h2>
+        <div class="recovery-codes">
```

Comment:
Are we missing an SCSS file? I can't find this class in master.

Reply:
I didn't actually define any new CSS here, just tried to put classes on it so someone who knows how to do that can fix my ugly work 😅